UPGRADING.md

```diff
@@ -32,178 +32,48 @@ API operations in `v1` of the Clerk Go SDK are organized by service. There are
 different services for each API and all services are properties of a single
 `clerk.Client`.
 
-Let's see a concrete example. Assume that we want to create an organization
```

Comment:
To the reviewer: It might be easier to click "View file" from the top right menu and see the final outcome in preview mode.